content/shared/v3-core-plugins/_index.md

```diff
@@ -35,6 +35,22 @@ influxdb3 serve \
   --object-store [OBJECT_STORE_TYPE] \
   --plugin-dir /path/to/plugins
 ```
+Replace `/path/to/plugins` with the directory where you want to store your Python plugin files. All plugin files must be located in this directory or its subdirectories.
+
+### Configure distributed environments
```

Comment:
This mostly, though not strictly, applies to Enterprise. We may need to dupe this page and move them into core and enterprise product directories.

Reply:
We should use one of the new `show-in` or `hide-in` shortcodes that Scott deployed yesterday.